eo-maven-plugin/src/test/java/org/eolang/maven/PullMojoTest.java

```diff
@@ -88,11 +88,11 @@ void pullsSuccessfully(@TempDir final Path temp) throws IOException {
     void pullsFromProbes(@TempDir final Path temp) throws IOException {
         new FakeMaven(temp)
             .withProgram(
-                "+package org.eolang.custom",
-                "",
```

Comment:
@maxonfjvipon this empty line is forbidden now? Why did you remove it?

Reply:
@yegor256 empty line is mandatory there, I just replaced it with `\n` in the code